Auto find cur (#205)
metricly/metricly-docs: content/integrations/AWS-Integration/AWS-IAM-Installation.md

![another-aws-accnt](/images/AWS-IAM-Installation/another-aws-accnt.png)
6. Provide the **Account ID** from your CloudWisdom AWS integration. _Leave Require MFA unchecked_.
7. Select **Next: Permissions**.

## 5. Define Role Permissions  

There are four options available for this role: standard permissions, minimal monitoring permissions, minimal cost permissions, and master billing account permissions. A Minimal permission policy must be created _before_ being assigned to an IAM Role.

{{% notice tip %}}

> Virtana recommends opening a second browser tab to follow this section when creating minimal read-only policies. Once the policy is created, use the original tab to resume setup of the IAM role.

Reviewer: ```suggestion
We recommend opening a second browser tab to follow this section when creating minimal read-only policies. Once the policy is created, use the original tab to resume setup of the IAM role.
```

{{% /notice %}}

### Standard Permissions

Grants blanket read-only access to collect CloudWatch **performance metrics** and **billing files** from S3.

{{% expand "View Steps." %}}

Continuing from **Section 4**:

1. For Attach permission policies, add all of the following:
 - CostExplorerAPIReadOnly (**Filter policies** > **Customer Managed**)
 - ReadCostAndUsageReportDefinitions (**Filter policies** > **Customer Managed**)
 - AmazonMQReadOnlyAccess
 - ReadOnlyAccess
 ![customer-managed](/images/AWS-IAM-Installation/customer-managed.png)
2. Select **Next Step: Tags** and add any needed tags; **this is an optional step and you may skip it.**
3. Select **Next: Review**.
4. Add **Role Name**: `CloudWisdom`.
5. Select **Create Role**.  You are returned to IAM Roles in your AWS console.
6. Select the new role you have created.
7. Copy the **Role ARN** and move on to **Section 6**.
![role-arn](/images/AWS-IAM-Installation/role-arn.png)

{{% /expand %}}

### Minimal Monitoring Permissions

> Grants read-only access to collect CloudWatch **performance metrics** for all AWS services and their integrations.

Reviewer: ```suggestion
Grants read-only access to collect CloudWatch **performance metrics** for the AWS services CloudWisdom provides an integration.
```

```json
{

>   "Version": "2012-10-17",

Reviewer 2: Wasn't this policy for monitoring?

Author: I'm unsure. That policy also contains s3 permissions, but more broadly right?

Should i keep the old section and clarify them as two separate minimum perm policies? one based in cost and the other monitoring?

Reviewer 2: I'm not sure either to be honest. @rclagett what do you think?

Reviewer: It is for monitoring, which we still do. So the policy should be kept. If we want a third policy  - minimum for cost performance  metrics that would be EC2, EBS, RDS, S3, ELB, ASG - then we could do that.

The three would be minimal for monitoring, minimal for cost performance metrics, minimal for billing files.
